### ISSUE TITLE
Direct messages integration

### ISSUE THREAD

**Issue by dankonino** [ParkinT/bible-bot#2]
Hello I would like to do that if it's possible, that would be perfect ;) 
If you can tell me something about it I would be happy ;-)

**Comment by ParkinT** [comment]
Please provide a bit more explanation.  Do you mean the bible response would come **only** to the person who called on it and NOT be visible in the channel for others?

If that is the case, [the output](https://github.com/ParkinT/bible-bot/blob/master/bible.rb#L50) would need to be modified.  As explained in [the documentation](https://api.slack.com/slash-commands#responding_to_a_command), you set the `response_type:` to "Ephemeral".
And, of course, this should be configurable by the user.  In this particular bot I have not provided any "configuration" options but that is an intermediate challenge if you feel compelled to take it on.

Another way to approach this would be with a [Slack Command](https://api.slack.com/custom-integrations#invoke_external_services_with_slash_commands)

**Comment by dankonino** [comment]
This is also interesting, but I meant only integration to direct messages also as addition to channels.

**Comment by dankonino** [comment]
And one more question: Is it possible to choose e.g. King James Version ?

**Comment by ParkinT** [comment]
Presently, I rely on the `www.openbible.info` API to provide the data.
But I tried to construct this in a modular way (the 'bible-verse' class is independent) so it **could be** expanded.
You are welcome to take a stab at adding either of these great features you mentioned!!
Just Fork, Hack, then Pull Request.

**Comment by dankonino** [comment]
Okay thank you for information. I will dig into that, need to learn that ;) and also one thing could be also good and that's e.g. bible verse Proverbs 10:1-5

**Comment by ParkinT** [comment]
> that's e.g. bible verse Proverbs 10:1-5
Agreed.  That **was** part of my original intention (when I conceptualized the idea).  It began so difficult to build - while constructing the initial MVP - that I moved it ⬇️ on the priority list. 
If you want to tackle that I would be glad to help you along.  We could even Pair Program on it !!!

**Comment by dankonino** [comment]
Nice man ;) it will be pleasure to work with you. I am right now still in need of lot's of learning. If you can tell me where to start for let's say fresh graduate that would be cool, if looking on that side I was studying theoretical craap in Slovakia, I'm bachelor and left that to work on real projects :+1:
